### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by taoramo**
Correct port 8443 in Readme.

### CODE REVIEW

**Comment by madfcat** [comment]
Why do we need this change? I always use port 443.
For me, it's always `https://localhost` without explicit port.

I would also suggest to change this part (in `index.html`) to relative path
```
        const popup = window.open(
          "https://localhost:8443/api/oauth/redirect", // This should match your Django URL pattern
          "42 Authorization"
        );
```
        
like this: `"/api/oauth/redirect"`

**Comment by taoramo** [comment]
> Why do we need this change? I always use port 443. For me, it's always `https://localhost` without explicit port.
1) we want to run this on school machines and you need sudo for 443.
2) oauth callback is currently setup for 8443.
3) there's no reason to not use 8443.
> 
> I would also suggest to change this part (in `index.html`) to relative path
> 
> ```
>         const popup = window.open(
>           "https://localhost:8443/api/oauth/redirect", // This should match your Django URL pattern
>           "42 Authorization"
>         );
> ```
> 
> like this: `"/api/oauth/redirect"`
This makes sense.

**Comment by madfcat** [comment]
@taoramo I think I get your point. We need to change 443 to 8443 in nginx.conf and docker-compose.yml too.
I guess I could never make it run with 8443 on my machine.
And I have never run this project on the school computer.
I will give it a try now.

File: README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ Our journey to transend at Hive
 
 ## How to run
 
-https://localhost/ - this is the lobby view, where most of the interactions, login and user dashboards will happen. its equivalent to https://localhost/index.html
+https://localhost:8443/ - this is the lobby view, where most of the interactions, login and user dashboards will happen. its equivalent to https://localhost:8443/index.html
 
 ### Game
```
Comment (madfcat) [inline README.md]: We do not use game.html at all. Our project implements SPA.
The file `game.html` should be removed completely. I think it is some legacy stuff. Therefore, we do not even need this Game part in our README.md

**Review by madfcat — CHANGES_REQUESTED** [review]

**Comment by madfcat** [comment]
> @taoramo I think I get your point. We need to change 443 to 8443 in nginx.conf and docker-compose.yml too. I guess I could never make it run with 8443 on my machine. And I have never run this project on the school computer. I will give it a try now.

@dnapi @nikkxll @taoramo
I successfully ran it with 8443 port. Should we stick to the port 8443 and 443 and just change it finally to 8443 in nginx.conf and docker-compose.yml?

I see it is already changed in:
https://github.com/DaveeHorvath/ascension/pull/99

**Review by madfcat — APPROVED** [review]

**Comment by dnapi** [comment]
I have also run it on 8443 on my laptop. both option should work for me

**Pull request merged**